### DISCUSSION TITLE
Notes Export

### DISCUSSION THREAD

**Discussion by Freewolf2000** [vigoren/foundryvtt-simple-calendar#197] [Q&A] [answered]
I know there is a way to export the calendar that I have set up. But is there a way to export the notes that I enter into a calendar? I do not seem to see a way to be able to do that. Am I missing something?

Thanks

Phillip

**Comment by vigoren** [comment, marked as answer]
Hey!

Currently there isn't a way to export notes but this is planned to be added. The plans are listed under 1.4.x project.

I'm just finishing up the 1.3.x project and will be starting 1 4.x fairly soon.

**Reply by zero127** [reply]
is there a way until this update happens that i can manually move notes? I have a need to remake the world from my current game and have hundreds of historical events on the calendar. Even if its just something to reference ad DM. Any help would be stellar. Tyvm either way. Wonderful module regardless.

**Reply by vigoren** [reply]
Hey, with version 2 of Simple Calendar you can export the notes. More details on how the exporting and importing of calendars and their notes can be found here https://simplecalendar.info/pages/docs/global-configuration/import-export.html

**Reply by zero127** [reply]
Ah! I wasn't on version 2. That's why I was confused. Using the json link I got it updated in foundry! Thanks so much. Also holy wow there's features galore in V2! <3

**Comment by Freewolf2000** [comment]
Awesome, thank you very much for the answer.
This calendar is really spectacular.
